WordPress/Classes/Utility/Analytics/WPAppAnalytics.m

```diff
@@ -338,14 +338,18 @@ + (void)track:(WPAnalyticsStat)stat withProperties:(NSDictionary *)properties {
     [WPAnalytics track:stat withProperties:properties];
 }
 
-+ (void)track:(WPAnalyticsStat)stat error:(NSError * _Nonnull)error {
++ (void)track:(WPAnalyticsStat)stat error:(NSError * _Nonnull)error withBlog:(Blog *)blog {
     NSError *err = [self sanitizedErrorFromError:error];
     NSDictionary *properties = @{
                                  @"error_code": [@(err.code) stringValue],
                                  @"error_domain": err.domain,
                                  @"error_description": err.description
     };
-    [self track:stat withProperties: properties];
+    [self track:stat withProperties: properties withBlogID:blog.dotComID];
```

Comment:
(Nit) I'd suggest passing only the required parameter – `dotComID` – in the `track` method. It's safer because managed objects have to be thread-confined, so it's generally best to avoid using them if possible.

Reply:
Thank you for the note and explanation Alex! I've addressed this in https://github.com/wordpress-mobile/WordPress-iOS/pull/22328/commits/979ee185118ed85e5b7418e6428f91a038aad3cc 🙇‍♀️